pkg/performanceprofile/controller/performanceprofile_controller_hypershift.go

```diff
@@ -0,0 +1,594 @@
+/*
+
+
```

Comment:
Since this was cut from performanceprofile_controller.go a while ago  lets do a revisit/compare to see if we are fully aligned with the latest

Reply:
In general I have tried to reuse as much code as possible so any change in the performance_profile controller's way of doing things will affect also the PP hypershift controller, but I will go through the last changes in PP to see if there is anything to add in hypershift.